UI/data/locale/en-US.ini

```
@@ -1034,7 +1034,8 @@ Basic.Settings.Output.Simple.Warn.Encoder="Warning: Recording with a software en
 Basic.Settings.Output.Simple.Warn.Lossless="Warning: Lossless quality generates tremendously large file sizes! Lossless quality can use upward of 7 gigabytes of disk space per minute at high resolutions and framerates. Lossless is not recommended for long recordings unless you have a very large amount of disk space available. Replay buffer is unavailable when using lossless quality."
 Basic.Settings.Output.Simple.Warn.Lossless.Msg="Are you sure you want to use lossless quality?"
 Basic.Settings.Output.Simple.Warn.Lossless.Title="Lossless quality warning!"
-Basic.Settings.Output.Simple.Encoder.Software="Software (x264)"
+Basic.Settings.Output.Simple.Encoder.Software.OpenH264.H264="Software (OpenH264)"
+Basic.Settings.Output.Simple.Encoder.Software.X264.H264="Software (x264)"
```

Avoid having to re-translate because the key is changed, plus OpenH264 will only provide H264.
```suggestion
Basic.Settings.Output.Simple.Encoder.Software.OpenH264="Software (OpenH264)"
Basic.Settings.Output.Simple.Encoder.Software.X264="Software (x264)"
```